request_raw.go

```
@@ -0,0 +1,552 @@
+		} else if queryKey := fieldVT.Tag.Get("query"); queryKey != "" {
+			value := reflectToQueryString(fieldVV)
+			sep := fieldVT.Tag.Get("join_sep")
+			if sep != "" {
+				query.Add(queryKey, strings.Join(value, sep))
+			} else {
+				for _, v := range value {
+					query.Add(queryKey, v)
+				}
+			}
+		} else if j := fieldVT.Tag.Get("json"); j != "" {
+			if strings.HasSuffix(j, ",omitempty") {
```

golangci-lint (1.64.8)

S1017: should replace this if statement with an unconditional strings.TrimSuffix

(gosimple)